navigation.md

```diff
@@ -18,7 +18,7 @@
   * [Step 3 - Markdown and Fork Tutorial](pages/vi/githubandmarkdown.md)
   * [Step 4 - Docker Tutorial](pages/vi/dockertutorial.md)
   * [Step 5 - Keeping Fork Updated](pages/vi/gitrepositories.md)
-  * [Step 6 - GitHub Issues Tutorial](pages/vi/firststeps.md#Step_6_-_Github_Issues_Tutorial)
+  * [Step 6 - GitHub Issues Tutorial](pages/vi/firststeps.md#Step_6_-_GitHub_Issues_Tutorial)
```

Comment:
Can you change Step_6_-_GitHub_Issues_Tutorial to github-issues.md ?

since when we clicking the dropdown menu, the users should be taken to the GitHub Issues page.

Reply:
@pattanawadee88 Done! The link now goes to github-issues.md